scripts/install_sshnp

```diff
@@ -0,0 +1,268 @@
+  echo "  -r, --repo <path>                      Install using local repo"
+  echo "      --help                             Display this help message"
+}
+
+parse_args() {
+  while [ $# -gt 0 ]; do
+    case "$1" in
+    --help)
+      usage
+      exit 0
+    ;;
+    -l|--local)
+      if [ $# -lt 0 ]; then
+        echo "Missing argument for $1";
+        exit 1;
+      fi
+      SSHNP_LOCAL="$2"
+      shift 2
+    ;;
+    -r|--repo)
+      if [ $# -lt 0 ]; then
+        echo "Missing argument for $1";
+        exit 1;
+      fi
+      SSHNP_DEV_MODE="$2"
+      shift 2
+    ;;
+    -c|--client)
+      if [ $# -lt 0 ]; then
+        echo "Missing argument for $1";
+        exit 1;
+      fi
+      CLIENT_ATSIGN="$2"
+      shift 2
+    ;;
+    -d|--device)
+      if [ $# -lt 0 ]; then
+        echo "Missing argument for $1";
+        exit 1;
+      fi
+      DEVICE_MANAGER_ATSIGN="$2"
+      shift 2
+    ;;
+    -h|--host)
+      if [ $# -lt 0 ]; then
+        echo "Missing argument for $1";
+        exit 1;
+      fi
+      HOST_RENDEZVOUS_ATSIGN="$2"
+      shift 2
+    ;;
+    *)
+      echo "Unknown argument: $1"
+      exit 1
+    ;;
+    esac
+  done
+
+  while [ -z "$CLIENT_ATSIGN" ]; do
+    read -rp "Client address (e.g. @alice_client): " CLIENT_ATSIGN;
+  done
+
+  while [ -z "$DEVICE_MANAGER_ATSIGN" ]; do
+    read -rp "Device address (e.g. @alice_device): " DEVICE_MANAGER_ATSIGN;
+  done
+
+  if [ -z "$HOST_RENDEZVOUS_ATSIGN" ]; then
+    echo Pick your default region:
+    echo "  am: Americas"
+    echo "  ap: Asia Pacific"
+    echo "  eu: Europe"
+    read -rp "> " HOST_RENDEZVOUS_ATSIGN;
+  fi
+
+  while [[ "$HOST_RENDEZVOUS_ATSIGN" != @rv_* ]]; do
+    case "$HOST_RENDEZVOUS_ATSIGN" in
+    [Aa][Mm]*)
+      HOST_RENDEZVOUS_ATSIGN="@rv_am"
+      ;;
+    [Ee][Uu]*)
+      HOST_RENDEZVOUS_ATSIGN="@rv_eu"
+      ;;
+    [Aa][Pp]*)
+      HOST_RENDEZVOUS_ATSIGN="@rv_ap"
+      ;;
+    esac
+    if [[ "$HOST_RENDEZVOUS_ATSIGN" != @rv_* ]]; then
+      echo "Invalid region: $HOST_RENDEZVOUS_ATSIGN"
+      read -rp "region: " HOST_RENDEZVOUS_ATSIGN;
+    fi
+  done
+
+  norm_atsign CLIENT_ATSIGN
+  norm_atsign DEVICE_MANAGER_ATSIGN
+  echo;
+}
+
+parse_env() {
+  if [ -z "$SSHNP_USER" ]; then
+    SSHNP_USER="$USER";
+  fi
+  HOME_PATH=$(eval echo "~$SSHNP_USER");
+
+  if [ -z "$SSHNP_VERSION" ]; then
+    SSHNP_VERSION="latest";
+  fi
+  URL="https://api.github.com/repos/atsign-foundation/sshnoports/releases/$SSHNP_VERSION";
+
+  case "$(uname)" in
+  Darwin)
+    PLATFORM="macos"
+    EXT="zip"
+    ;;
+  Linux)
+    PLATFORM="linux"
+    EXT="tgz"
+    ;;
+  *)
+    PLATFORM="Unknown"
+    ;;
+  esac
+
+  if [ "$PLATFORM" == "Unknown" ]; then
+    echo "Unsupported platform: $(uname)";
+    exit 1;
+  fi
+
+  # ARCH includes the dot at the end to avoid conflict between arm and arm64
+  case "$(uname -m)" in
+  x86_64)
+    ARCH="x64."
+    ;;
+  aarch64|arm64)
+    if [ "$PLATFORM" == "macos" ]; then
```

Comment:
I thought @cconstab was going to fix the MacOS binaries to properly follow the [naming convention](https://github.com/atsign-foundation/operations/blob/trunk/decisions/2023-03-dart-naming-convention-for-releases.md)

Reply:
Yes, we won't need this for next release, but to test it against this release we do. This will match both "arm64" and "arm" for macos.